src/cryptonote_core/loki_name_system.cpp

```diff
@@ -1056,7 +1292,7 @@ CREATE TABLE IF NOT EXISTS "mappings" (
     "txid" BLOB NOT NULL,
     "prev_txid" BLOB NOT NULL,
     "register_height" INTEGER NOT NULL,
-    "update_height" INTEGER NOT NULL,
+    "update_height" INTEGER NOT NULL DEFAULT "register_height",
```

Comment:
This is a bug originally in the previous PR, in that when you alter a table you append the new column to the table at the end.

So then when you go and query the row and try to index into it using the enum (that specifies the wrong index on migrated DB's), it gives you the wrong value.

Reply:
Reordered the columns so new and migrated will match.  (You'll have to dump your intermediate sqlite db if you recreated one from scratch with the previous PR).